### DISCUSSION TITLE
Can't run model with Lipophilicity value > 10

### DISCUSSION THREAD

**Discussion by jonathanlsn** [Open-Systems-Pharmacology/Forum#798] [Questions & Problems] [answered]
Hello,

I recently met an issue with the logP I have to set in compound parameters on PK-sim. I wanted to set this value to 11.2 but the software return this error "Value for lipophilicity should be less than or equal to 10 Log Units". 
I would like to know why a value higher than 10 can't be handled by the software ?

Thank you in advance

**Comment by tobiasK2001** [comment]
Dear Jonathan,

why do you think your compound could hav a logP of 11.2? PK-Sim sets a limit here because a logP > 10 would be well outside the accepted possible ADMET space of NCEs. See here for more details
https://pubmed.ncbi.nlm.nih.gov/11259830/

best, Tobias

**Comment by StephanSchaller** [comment, marked as answer]
@jonathanlsn, extending the comment ba @tobiasK2001,  the reason for PK-Sim to limit this range is related to "model confidence".
The partitioning methods contained in PK-Sim are developed for pharmaceutical NCEs where the "Rule of 5" referred to by @tobiasK2001 are mostly adhered to (LogP < 5 is one of these rules).
This means that the PK-Sim predicted partitioning based on a LogP>5 might be wrong, as the partition coefficient methods available in PK-Sim are not "qualified" for such compounds/LogP ranges.
We have worked with a couple of compounds (organic chemicals) with LogP > 5 (but < 10) and looked into other partitioning methods (e.g. Poulin & Haddad) to better characterize the ADME, but there are quite a few additional aspects that make things complicated for highly lipophilic chemicals.
